store/copr/coprocessor.go

```diff
@@ -1637,15 +1637,15 @@ func (worker *copIteratorWorker) handleCollectExecutionInfo(bo *Backoffer, rpcCt
 	td := util.TimeDetail{}
 	if pbDetails := resp.pbResp.ExecDetailsV2; pbDetails != nil {
 		// Take values in `ExecDetailsV2` first.
-		if timeDetail := pbDetails.TimeDetail; timeDetail != nil {
```

Comment:
maybe can just put a line at 1637
td.MergeFromTimeDetail(resp.pbResp.ExecDetailsV2.GetTimeDetailV2(), resp.pbResp.ExecDetails.GetTimeDetail())

Reply:
No we can't. It's possible ExecDetailsV2.TimeDetail != nil but ExecDetailsV2.TimeDetailV2 and ExecDetails.TimeDetail is nil.